### PR TITLE
VPN hero image A/B testing

### PR DESCRIPTION
**Pull request by reemhamz**
## Description
Create 4 different variations of the VPN page to reflect the Be You Or Blur You campaign:

- Variation reflecting current page
- Variation with updated English image
- Variation with updated French image
- Variation with updated German image

## Issue / Bugzilla link
#10505 
Original PR (that got botched): https://github.com/mozilla/bedrock/pull/10530

## Testing
Current page
http://localhost:8000/en-US/products/vpn/?entrypoint_experiment=vpn-landing-page-image-change&entrypoint_variation=current

Variations page. Change the locale to get the different images (make sure to use a browser without do-not-track enabled)
http://localhost:8000/en-US/products/vpn/?entrypoint_experiment=vpn-landing-page-image-change&entrypoint_variation=blur

### CODE REVIEW

File: bedrock/products/tests/test_views.py.orig [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```
[the remark this answers is not in the document]
Comment (reemhamz) [reply bedrock/products/tests/test_views.py.orig:1]: Ah, was wondering what those were. Thanks for this!

File: bedrock/products/templates/products/vpn/includes/macros.html [hunk 2]
```diff
@@ -22,37 +22,72 @@ <h2 class="vpn-hero-sub-heading">{{ subheading }}</h2>
...
+        <h2 class="vpn-hero-sub-heading">{{ subheading }}</h2>
+        <div class="vpn-hero-desc">
+          <p>{{ desc }}</p>
+          {{ caller() }}
+        </div>
+      </div>
+
+      <div class="vpn-hero-blur-image">
+        {% if LANG.startswith('en-') %}
+          <img src="{{ static('img/products/vpn/landing/wnp-93-vpn-hero-en.jpg') }}" alt="">
```
[the remark this answers is not in the document]
Comment (reemhamz) [reply bedrock/products/templates/products/vpn/includes/macros.html:79]: Weird. I already optimized them and double checked, but thanks for the heads up, i'll do it again :) !

[the remark this answers is not in the document]
Comment (alexgibson) [reply bedrock/products/templates/products/vpn/includes/macros.html:79]: I think the images are also quite a bit larger than the dimensions in which they are displayed on the page

File: bedrock/products/tests/test_views.py [hunk 3]
```diff
@@ -95,3 +95,35 @@ def test_vpn_landing_page_variant_b_template(self, render_mock):
...
+    def test_vpn_landing_page_variant_current_template(self, render_mock):
+        req = RequestFactory().get("/products/vpn/?entrypoint_experiment=vpn-landing-page-image-change&entrypoint_variation=current")
+        req.locale = "en-US"
+        view = views.vpn_landing_page
+        view(req)
+        template = render_mock.call_args[0][1]
+        assert template == "products/vpn/variations/hero-image-current.html"
+
+    def test_vpn_landing_page_variant_en_template(self, render_mock):
+        req = RequestFactory().get("/products/vpn/?entrypoint_experiment=vpn-landing-page-image-change&entrypoint_variation=en")
```
[the remark this answers is not in the document]
Comment (reemhamz) [reply bedrock/products/tests/test_views.py:108]: @alexgibson so should I keep the 3 locales tests and change the `entrypoint_variation` to be `=blur`? That way we can keep the tests for each locale we're testing this on, focusing the locale on the `req.locale` attribute.

**Comment by reemhamz** [comment]
Ticket basically completed. Awaiting to receive the updated images for the a/b test.

**Comment by alexgibson** [comment]
Heads up that this needs a rebase ^. This branch may also need a further rebase should https://github.com/mozilla/bedrock/issues/10433 merge first.

**Comment by reemhamz** [comment]
Note: rebase and add `isApprovedToRun` [function ](https://bedrock.readthedocs.io/en/latest/abtest.html?highlight=isapproved#avoiding-experiment-collisions)when ticket is ready to continue.

**Comment by reemhamz** [comment]
Closed due to campaign ending

**Pull request closed without merging**